customer/client_test.go

```diff
@@ -159,18 +159,22 @@ func TestCustomerGet(t *testing.T) {
 func TestCustomerDel(t *testing.T) {
 	res, _ := New(nil)
 
-	err := Del(res.ID)
+	c, err := Del(res.ID)
```

Comment:
From what I can tell, it doesn't seem too idiomatic in these to use single letter variable names in places like this (also, `c` is very often used in this project to refer to a client). Do you think we should rename this to something like `resDel`?